### DISCUSSION TITLE
Only load S3 with ZIP method

### DISCUSSION THREAD

**Discussion by ghost** [aws/aws-sdk-php#2759] [Q&A] [answered]
I'm using the SDK witht the zip method, this one ( https://github.com/aws/aws-sdk-php/issues/808 ) is about the composer way. How do I only load the specific services like S3 only instead of loading everything?

**Comment by yenfryherrerafeliz** [comment]
Hi @karchung0930, could you please try with the following feature [here](https://github.com/aws/aws-sdk-php/tree/master/src/Script/Composer), which is basically aimed to keep just the services that you have set to be used.

Please let me know if that helps!

Thanks!

**Comment by ghost** [comment]
@yenfryherrerafeliz Hi, your reference is exactly the one I have mentioned at the first place which is for composer. But I'm using the SDK with the zip method: https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/getting-started_installation.html#installing-by-using-the-zip-file

**Comment by yenfryherrerafeliz** [comment, marked as answer]
Hi @karchung0930,  unfortunately this feature is only available for composer projects.

[a comment by an automated account is not shown]
